RijsMuseum/app/src/main/java/com/example/rijsmuseum/fragments/ArtifactsFragment.kt

```diff
@@ -47,14 +48,9 @@ class ArtefactsFragment : Fragment() {
         adapter.setOnItemClickListener(object : RecyclerAdapter.OnClickCallback {
             override fun onItemClick(cList: Collections.ArtObject) {
                 requireActivity().supportFragmentManager.beginTransaction().apply {
```

Comment:
If you are using the navigation component, why do you need the fragment transaction?